src-vue/src/plugins/store.ts

```diff
@@ -334,6 +346,24 @@ export const store = createStore<FlightCoreStore>({
                 i18n.global.tc(`channels.names.${state.northstar_release_canal}`),
                 i18n.global.tc('channels.release.switch.text', {canal: state.northstar_release_canal}),
             );
+        },
+        async fetchProfiles(state: FlightCoreStore) {
+            let game_install = {
+                game_path: state.game_path,
+                profile: state.active_profile,
+                install_type: state.install_type
+            } as GameInstall;
```

Comment:
One thing I wanted to do forever but always shyed from (cause it was kinda boring work) was to put the entire GameInstall object in the store as opposed to the individual struct members.

Not sure if that's something we wanna do before adding profile support of after given the it modifies the `GameInstall` struct. (obv separate PR either way). Thoughts?

Reply:
Would simplify a lot of code and the game install storage.
I'd say before, I can slap together a PR that deals with it all.